src/app/mock/demo-target-environment.json

```diff
@@ -4,17 +4,17 @@
 
   "id": "os",
   "styleClass": "card-pf-footer--logo-os"
 }, {
   "benefits": [
     "Scaffolding for your new application is generated.",
-    "Download the project as a ZIP file and manually deploy your application to another OpenShift instance."
+    "Download the project as a ZIP file and manually deploy your application to an OpenShift instance."
```

Comment:
This is a 'demo' of the target environment json... the text in the actual app is:
`The project contains the setup and instructions to manually deploy the application in any OpenShift instance.`

https://github.com/fabric8-launcher/launcher-frontend/blob/94be10eff229ccf596a4893e38558e60eb507015/src/app/wizard/services/app-launcher-target-environment.service.ts#L33-L36